metsrw/plugins/premisrw/premis.py

```diff
@@ -490,6 +491,8 @@ def _data_to_lxml_el(data, ns, nsmap, element_maker=None, snake=True):
                 element, ns, nsmap, element_maker=element_maker, snake=snake))
         elif isinstance(element, six.text_type):
             args.append(element)
+        elif isinstance(element, etree._Element):
```

Comment:
I don't like this check (`_Element`), but i haven't found a better way.

Reply:
It's good enough. My feeling is that the naming convention in lxml relates to some weird convention in CPython or maybe C (libxml)?